packages/frontend/src/components/DashboardTiles/TileForms/TileUpdateModal.tsx

```diff
@@ -58,18 +75,27 @@ const TileUpdateModal = <T extends Tile>({
             onClose={handleClose}
             backdropClassName="non-draggable"
         >
-            <Form
-                name="Edit tile content"
-                methods={form}
-                onSubmit={handleConfirm}
-            >
+            <form onSubmit={handleConfirm}>
                 <DialogBody>
                     {tile.type === DashboardTileTypes.SAVED_CHART ? (
                         <ChartTileForm />
                     ) : tile.type === DashboardTileTypes.MARKDOWN ? (
-                        <MarkdownTileForm />
+                        <MarkdownTileForm
+                            form={
+                                form as UseFormReturnType<
+                                    DashboardMarkdownTileProperties['properties']
+                                >
```

Comment:
Hi Magnew, Thanks for the review, I am not sure if `Hide Title` is needed for Markdown., Since it is excluded in the current implementation, I didn't implement it. 
Yes, if it is needed for Mardown, I could do it in a separate PR !

Reply:
It actually looks like the markdown tiles accept an empty title and just don't render it. So we don't need to implement that -- there's a way to get the behavior.

Reply:
I'm sorry, I didn't get the question. 
Markdown Tile doesn't have `hideTitle` as one of its property ( [DashboardMarkdownTileProperties](https://github.com/lightdash/lightdash/blob/main/packages/common/src/types/dashboard.ts#L26) )
Yes, if it is needed we could extend this property to make it. 

Not sure if I understood it correctly, if its needed, please make an issue and I would be happy to take a look :)